src/core/client/ui/components/Button/Button.css

```diff
@@ -573,3 +580,33 @@
     }
   }
 }
+
+.variantTextUnderlined {
+  font-weight: 700;
+  &:not(.disabled) {
+    &.colorRegular {
```

Comment:
Hmm how changing this to `.colorPrimary`? 
And have `.colorRegular` to not change colors (e.g. appear grey).
Following the other variants, regular is grey and only primary gets the primary color.

Reply:
Obviously the buttons would need to be changed to have color="primary" then.